package-lock.json

```diff
@@ -0,0 +1,4502 @@
+{
```

Comment:
Please don't check in `package-lock.json`.  We use yarn instead of npm, so only `yarn.lock` should be checked in (when appropriate).